<ui version="4.0" >
 <class>QgsComposerMapWidgetBase</class>
 <widget class="QWidget" name="QgsComposerMapWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>374</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Map options</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Map</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel1" >
        <property name="text" >
         <string>Width</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="buddy" >
         <cstring>mWidthLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLineEdit" name="mWidthLineEdit" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="textLabel2" >
        <property name="text" >
         <string>Height</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="buddy" >
         <cstring>mHeightLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLineEdit" name="mHeightLineEdit" />
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Scale</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>1:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLineEdit" name="mScaleLineEdit" >
        <property name="inputMask" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QGroupBox" name="mMapExtentGroupBox" >
     <property name="title" >
      <string>Map extent</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="mXMinLabel" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>X min</string>
        </property>
        <property name="buddy" >
         <cstring>mXMinLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="mXMinLineEdit" />
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="mXMaxLabel" >
        <property name="text" >
         <string>X max</string>
        </property>
        <property name="buddy" >
         <cstring>mXMaxLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="2" >
       <widget class="QLineEdit" name="mXMaxLineEdit" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="mYMinLabel" >
        <property name="text" >
         <string>Y min</string>
        </property>
        <property name="buddy" >
         <cstring>mYMinLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="mYMinLineEdit" />
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="mYMaxLabel" >
        <property name="text" >
         <string>Y max</string>
        </property>
        <property name="buddy" >
         <cstring>mYMaxLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2" >
       <widget class="QLineEdit" name="mYMaxLineEdit" />
      </item>
      <item row="2" column="0" colspan="4" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>311</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="4" >
       <widget class="QPushButton" name="mSetToMapCanvasExtentButton" >
        <property name="text" >
         <string>Set to map canvas extent</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="mPreviewModeLabel" >
     <property name="text" >
      <string>Preview</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="buddy" >
      <cstring>mPreviewModeComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QComboBox" name="mPreviewModeComboBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QPushButton" name="mUpdatePreviewButton" >
     <property name="text" >
      <string>Update preview</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QCheckBox" name="mKeepLayerListCheckBox" >
     <property name="text" >
      <string>Keep layer list</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>458</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>mWidthLineEdit</tabstop>
  <tabstop>mHeightLineEdit</tabstop>
  <tabstop>mScaleLineEdit</tabstop>
  <tabstop>mXMinLineEdit</tabstop>
  <tabstop>mXMaxLineEdit</tabstop>
  <tabstop>mYMinLineEdit</tabstop>
  <tabstop>mYMaxLineEdit</tabstop>
  <tabstop>mSetToMapCanvasExtentButton</tabstop>
  <tabstop>mPreviewModeComboBox</tabstop>
  <tabstop>mUpdatePreviewButton</tabstop>
  <tabstop>mKeepLayerListCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
